blog/zh-cn/Eavy_Info.md

```diff
@@ -41,58 +44,64 @@ DS 作为调度系统支持以下功能：
 
 回归业务本身，我们的业务场景，数据同步的业务需要是类型多，但数据量基本不会特别大，对实时要求并不高。所以在架构选型之初，我们就选择了 datax+ds 的组合，并进行对应业务的改造实现，现在作为服务产品融合在各个项目中，提供离线同步服务。
 
+<br />
```

Comment:
Why we need these? Line breaks in markdown should be rendered as `<br/>` in html, using too many html tags in markdown is not a good choice, I'd avoid to do so

Reply:
The picture is too close to the text, so < br / > is used for line feed

In order to be more beautiful, some adjustments have been made to this.

Should HTML syntax not be used?

Reply:
> Should HTML syntax not be used?

It can be used when really needed, as you know, markdown is created to let people focus more on the content and less on the style, if we have too many HTML tags in a markdown file, it makes no sense to use markdown. It's not a blocker for this PR, I just want to raise the concern for maintainers to notice this: don't overuse HTML in markdown, otherwise we should consider just use `.html` files instead of markdown

Reply:
ok

Reply:
Ok, thanks